src/engine/test/source/builder/builders/opBuilderHelperStringFromArray_test.cpp

```diff
@@ -0,0 +1,134 @@
+/* Copyright (C) 2015-2021, Wazuh Inc.
+ * All rights reserved.
+ *
+ * This program is free software; you can redistribute it
+ * and/or modify it under the terms of the GNU General Public
+ * License (version 2) as published by the FSF - Free Software
+ * Foundation.
+ */
+
+#include <any>
+#include <gtest/gtest.h>
+#include <vector>
+
+#include <baseTypes.hpp>
+
+#include "opBuilderHelperMap.hpp"
+
+using namespace base;
+namespace bld = builder::internals::builders;
+
+TEST(opBuilderHelperStringFromArray, Builds)
+{
+    auto tuple = std::make_tuple(std::string {"/field"},
```

Comment:
Please, add the `const` modifier to every variable that won't change its value. If you have values that are repeated many times you could define a `constexpr` or `const string`, or similar, to avoid redundancy of code.

Reply:
done